docs/ModelDescriptors.toml

```diff
@@ -69,9 +69,9 @@ FactorAnalysis_MultivariateStats = ["dimension_reduction", ]
 FeatureAgglomeration_MLJScikitLearnInterface = ["clustering", "static_models", "feature_engineering"]
 FeatureSelector_FeatureSelection = ["dimension_reduction", "feature_engineering"]
 FillImputer_MLJModels = ["missing_value_imputation", ]
-GaussianMixtureClusterer_BetaML = ["clustering", "distribution_fitter"]
-GaussianMixtureImputer_BetaML = ["missing_value_imputation", "distribution_fitter"]
-GaussianMixtureRegressor_BetaML = ["regression", "distribution_fitter"]
+GaussianMixtureClusterer_BetaML = ["clustering", "distribution_estimation"]
```

Comment:
Just making sure, you did intend to call it "distribution estimation" instead of "density estimation", right? Because the latter may be more common.

Reply:
oh bother. I did mean to make it "density estimation". 

good catch.